### PR TITLE
Mass effect

### PR DESCRIPTION
**Pull request by mlooney**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,16 +9,19 @@ jobs:
...
           DATABASE_NAME: circle_test
+          POSTGRES_DB: circle_test
           DATABASE_USERNAME: postgres
-          POSTGRES_DB: tenejo_test
           POSTGRES_HOST: 127.0.0.1
           POSTGRES_USER: postgres
           WORKING_PATH: /tmp
           UPLOAD_PATH: /tmp
           CACHE_PATH: /tmp/cache
+          CIRCLE_FEDORA_URL: http://127.0.0.1:8080/fcrepo/rest
```
Comment (maxkadel) [inline .circleci/config.yml:23]: I don't think the CIRCLE_FEDORA and CIRCLE_SOLR variables should be necessary? they aren't needed on main

File: config/blacklight.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
+  url: http://solr:8983/solr/hydra-development
 test: &test
   adapter: solr
-  url: <%= ENV['SOLR_TEST_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_TEST_PORT', 8985)}/solr/hydra-test" %>
+  url: <%= ENV['CIRCLE_SOLR_URL'] || 'http://solr:8983/solr/hydra-test' %>
```
Comment (maxkadel) [inline config/blacklight.yml:6]: I don't think this should be needed, just add `SOLR_TEST_URL: http://solr:8983/solr/hydra-test` to the web & sidekiq environments in docker-compose.yml

File: config/blacklight.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
+  url: http://solr:8983/solr/hydra-development
```
Comment (maxkadel) [inline config/blacklight.yml:3]: I think this is going to break things in non-dockerized development - I'd just add these to the docker-compose.yml web environment, and I believe it should work in CI, docker-dev, and non-docker-dev:

```
      SOLR_URL: http://solr:8983/solr/hydra-development
      SOLR_TEST_URL: http://solr:8983/solr/hydra-test
```

File: config/fedora.yml [hunk 4]
```diff
@@ -1,12 +1,13 @@
 development:
   user: fedoraAdmin
   password: fedoraAdmin
-  url: <%= ENV['FEDORA_DEV_URL'] || "http://127.0.0.1:#{ENV['FCREPO_DEVELOPMENT_PORT'] || 8984}/rest" %>
+  url: http://fedora:8080/rest
```
Comment (maxkadel) [inline config/fedora.yml:4]: same as solr - add this to docker-compose.yml web environment:

```
      FEDORA_HOST: http://fedora
      FEDORA_PORT: 8080
      FEDORA_DEV_URL: http://fedora:8080/rest
      FEDORA_BASE_PATH: /dev
      FEDORA_TEST_URL: http://fedora:8080/rest
      FEDORA_TEST_BASE_PATH: /test
```

File: config/solr.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # This is a sample config file that points to a solr server for each environment
 development:
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV['SOLR_TEST_PORT'] || 8983}/solr/hydra-development" %>
+  url: http://solr:8983/solr/hydra-development
```
Comment (maxkadel) [inline config/solr.yml:3]: same as other solr comments

**Review by maxkadel — CHANGES_REQUESTED** [review]

File: spec/system/create_work_spec.rb [hunk 6]
```diff
@@ -7,6 +7,8 @@
 # NOTE: If you generated more than one work, you have to set "js: true"
 RSpec.describe 'Create a Work', type: :system, clean: true, js: true do
   context 'a logged in user' do
+    # TODO: Remove following puts line once database issue has been resolved
```
Comment (maxkadel) [inline spec/system/create_work_spec.rb:10]: Boop!

**Review by mark-dce — APPROVED** [review]
Let's get these into main and try to do any further refinement in individual PRs

File: .circleci/config.yml (hunk 1 above)
Comment (mark-dce) [reply .circleci/config.yml:23]: We're going to merge what we have into main and then work in smaller chunks for cleanup as needed against main.

File: config/blacklight.yml (hunk 3 above)
Comment (mark-dce) [reply config/blacklight.yml:3]: Agreed, I'd like to get the changes into main and then work from there.

**Pull request merged**